src/test/scala/cognite/spark/v1/FlexibleDataModelCorePropertyRelationTest.scala

```diff
@@ -97,6 +97,7 @@ class FlexibleDataModelCorePropertyRelationTest
     val instanceExtIdNode = s"${randomId}Node"
     val instanceExtIdEdge = s"${randomId}Edge"
 
+    //scalastyle:off method.length
```

Comment:
this wasn't needed before.... did something change in the config?

Reply:
idk ^^ maybe the config now asks the tests to be formatted properly too?